app/javascript/components/batch_update/components/SearchForm.js

```diff
@@ -111,6 +113,7 @@ class SearchForm extends React.Component {
           onRemoveTag={onRemoveTag}
           onRemoveArtist={onRemoveArtist}
           partner={partner}
+          sale={sale}
```

Comment:
**suggestion:** This is on the right track, but we need to thread this a little further [and replicate this](https://github.com/artsy/rosalind/blob/8dac405e64f7d1164eb2b1b2211ae4decf10ebb7/app/javascript/components/batch_update/components/CurrentCriteria.js#L65) in order to have the current sale criterion (and dismiss button) actually show up in the UI.

Reply:
**suggestion:** add test coverage as well, [like so](https://github.com/artsy/rosalind/blob/8dac405e64f7d1164eb2b1b2211ae4decf10ebb7/app/javascript/components/batch_update/components/CurrentCriteria.spec.js#L80-L85).